src/components/fields/date-range-field/README.md

```diff
@@ -0,0 +1,59 @@
+# DateRangeField
+
+#### Description
+
+A controlled date input component for a date range.
+
+## Usage
+
+```js
+import { DateRangeField } from '@commercetools-frontend/ui-kit';
+
+<DateRangeField
+  title="Release Date"
+  value={['2018-09-20', '2018-09-24']}
+  onChange={event => alert(event.target.value)}
+/>;
+```
+
+#### Properties
+
+| Props                  | Type               | Required | Values                             | Default | Description                                                                                                                                                                                                                                                           |
+| ---------------------- | ------------------ | :------: | ---------------------------------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`                   | `string`           |    -     | -                                  | -       | Used as HTML `id` property. An `id` is auto-generated when it is not specified.                                                                                                                                                                                       |
+| `horizontalConstraint` | `object`           |          | `xs`, `s`, `m`, `l`, `xl`, `scale` | `scale` | Horizontal size limit of the input fields.                                                                                                                                                                                                                            |
```

Comment:
let's make our lives easier? #325

Reply:
I want to do that in one go at the end instead of having to think about it for every component now. I want to tackle it once for all fields instead of ending up with a mix of some components only offering the values they support while others still offer all values.

It's also easier to communicate that we didn't do this at all so far and will tackle it once later instead of trying to communicate that we only did it for some and not for others and then consumers can never be sure which ones are supported and which ones aren't.